<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HPEImageDialogForm</class>
 <widget class="QWidget" name="HPEImageDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>310</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#HPEImageDialogForm {
	background-color: rgb(255, 255, 255);
}

QGroupBox {
	background-color: rgb(255, 255, 255);
     border-radius: 10px;
	border: 4px solid rgb(208, 208, 208);
	color: gray;
	margin-top:  1ex; /* leave space at the top for the title */
}

QGroupBox::indicator {
    width: 24px;
    height: 24px;
}

QGroupBox::indicator:checked {
    image: url(:/icons/resources/icons/ico_checked.png);
}

QGroupBox::indicator:unchecked {
    image: url(:/icons/resources/icons/ico_unchecked.png);
}

QPushButton
{
	padding: 4px 12px;
     color: gray;
     background-color: white;
     border-style: solid;
     border-width: 5px;
     border-radius: 8px;
     border-color: rgb(208, 208, 208);
     font: bold 13px;
}

QPushButton:hover
{
	/*border-color: gray;*/
     color: rgb(255, 255, 255);
     background-color: rgb(208, 208, 208);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="addedImagesGroup">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>from Added Images</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QListWidget" name="imageList">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="newSourceGroup">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>from New Sources</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="sourceLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>File Path or URL :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border-bottom: 2px solid rgb(235, 235, 235);
}

QLineEdit:focus {
	border-bottom: 2px solid gray;
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openFileButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
